### ISSUE TITLE
sometimes meeting information is not avaiable in getMeetingInfo api

### ISSUE THREAD

**Issue by saeed-parsaee** [blindsidenetworks/scalelite#726]
**Describe the bug**

We occasionally see that the meeting information is **not avaiable** in **getMeetingInfo** api, although it is already exist in **getMeetings** api of scalelite.

FYI: meeting still running with attendees and moderator and we can join via api-mate or directly calling the bbb join api

***Deployment:***
1. Deployment:  It is based on the default systemd deployment
2. Versions: Currently 1.3 but issue happens on 1.2 and earlier. 

**Expected behavior**

Meeting data should be available while meeting is running

**Comment by amalrails** [comment]
@saeed-parsaee whats the error response you are getting? is it `We could not find a meeting with that meeting ID`? could you share a screenshot of the response

**Comment by saeed-parsaee** [comment]
@amalrails Yes. That's the what I get from getMeetingInfo api of scalelite.
I don't have any screenshot right now. but I will send it ASAP.

FYI:
1- Meeting data exist in getMeetings  from scalelie
2- Meeting data exist in getMeetingInfo  from BBB server
1- Meeting data exist in getMeetings  from BBB server

**Comment by jfederico** [comment]
The issue is that the getMeetingInfo which makes use of the redis database for its response, is being performed before the poller finished its update. And getRecordings which is based on the getRecordings response received from the actual BBB servers may be responding faster than the poller.

While this would work normally fine in small deployments, it won't scale. 

Bottom line the getRecordings response should be based on the info contained in the local redis database as well.

The poller(s) should be the only one(s) sending requests to the BBB servers for fetching data.

**Comment by harika1258** [comment]
Hi...
Sometimes I'd like to end an special meetings in the Scalelite topology.
With this command : ./bin/rake meetings:end["meetingId1:meetingId2:meetingId3"]
But we don't have meetingId1,2,3...
just we have meeting name and ID such as this : son-ypc-yxs-ryw 
API mate didn't response Meeting id,
How can we get meetingid as a GUID code?
Thanks.
